.github/workflows/deploy.yml

```diff
@@ -55,74 +54,96 @@ jobs:
            sudo mv s2i /usr/bin
            which s2i
            s2i version
+      - name: install helm and wildfly helm-charts
```

Comment:
for the installation of Helm, an alternative is to use another action:

```
      - uses: redhat-actions/openshift-tools-installer@v1.1
         id: install_clients
         with:
           helm: "3"
```

you still have to add the wildfly chart like you do

Reply:
@jmesnil , thank-you! much simpler. I fixed it.